HostApp/HostApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,8 +5,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/aws-amplify/amplify-swift",
       "state" : {
-        "revision" : "7d9640a5b7416f02f03463972dc2c6d87b295545",
-        "version" : "2.27.2"
+        "branch" : "no-light-implementation",
```

Comment:
revert branch

Reply:
i can revert it in a follow up as it's needed for CI to build the changes in this PR.